api/CSSStyleDeclaration.json

```
@@ -544,6 +544,55 @@
             "deprecated": false
           }
         }
+      },
+      "@@iterator": {
+        "__compat": {
+          "mdn_url": "https://developer.mozilla.org/docs/Web/API/CSSStyleDeclaration/@@iterator",
+          "support": {
+            "chrome": {
+              "version_added": "51",
+              "notes": "Before Chrome 51, we can iterate it after calling <code>Array.from()</code> to transfer it."
+            },
+            "chrome_android": {
+              "version_added": null
+            },
+            "edge": {
+              "version_added": null
+            },
+            "firefox": {
+              "version_added": "36"
+            },
+            "firefox_android": {
+              "version_added": null
+            },
+            "ie": {
+              "version_added": false
+            },
+            "opera": {
+              "version_added": "30"
+            },
+            "opera_android": {
+              "version_added": null
```

Confirmed that it works in the current Opera for Android, the version number based on the underlying engine version.

```suggestion
              "version_added": "41"
```